dlt/common/schema/typing.py

```diff
@@ -166,6 +198,7 @@ class TTableSchema(TypedDict, total=False):
     columns: TTableSchemaColumns
     resource: Optional[str]
     table_format: Optional[TTableFormat]
+    cdc_config: Optional[TCdcConfig]
```

Comment:
this is one way to go. but IMO a better way would be to define a column level hint.
`cdc_op` which could be integer or single char (u/d/i)

do we really need a sequence? if so we could reuse `sort` or add a new hint ie. `cdc_seq`. There are helper methods to find column(s) with hints

it looks simpler to me.